api/v1alpha1/releaseplan_types.go

```diff
@@ -17,20 +17,34 @@ limitations under the License.
 package v1alpha1
 
 import (
+	shared "github.com/redhat-appstudio/shared-utils/tekton"
```

Comment:
As I commented with you, the final version of the whole change was already implemented (the one including shared-utils) so I had to revert all of that. This I just simply forget about it. I restored the package and left the named imports so it's easy to go back to shared-utils later on.

Reply:
Then if, after review, we decide not to use shared-utils (as is my preference) we have all these imports changed to `shared` and it not making much sense. Why not do keep all the shared-utils stuff in one PR, instead of half now half later